core/build.gradle

```diff
@@ -108,6 +108,7 @@ dependencies {
     optionalApi libs.micrometer.prometheus
     optionalApi libs.dropwizard.metrics.core
     optionalApi libs.prometheus
+    api libs.context.propagation
```

Comment:
Should it be `api` or `implementation`? If we move `RequestContextAccessor` to `internal.common`, then we can probably keep it as `implementation`.

Reply:
Regarding this https://github.com/line/armeria/pull/5577#discussion_r1577008349,
we can use `implementation ` because i move `RequestContextThreadLocalAccsoor` to `internal` packages.

Reply:
Then let's use `implementation` :bow:

Reply:
Thanks for your comment. 🙇‍♂️ 
i missed it! i modified it.